src/DynamoCore/Graph/Annotations/AnnotationModel.cs

```diff
@@ -498,9 +500,19 @@ internal void UpdateBoundaryFromSelection()
 
                 this.X = region.X;              
                 this.Y = region.Y;
-                this.Width = Math.Max(region.Width, TextMaxWidth + ExtendSize);
                 this.ModelAreaHeight = IsExpanded ? region.Height : ModelAreaHeight;
-                this.Height = this.ModelAreaHeight + TextBlockHeight;
+                Height = this.ModelAreaHeight + TextBlockHeight;
+
+                if (IsExpanded)
+                {
+                    Width = Math.Max(region.Width, TextMaxWidth + ExtendSize);                    
+                    lastExpandedWidth = Width;
+                }
+                else
+                {
+                    //If the annotation is not expanded, than it will remain the same width of the last time it was expanded
```

Comment:
~~than~~ -> then
`expanded, then it will `

Reply:
whops, thanks!